doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -15,13 +17,13 @@ The most relevant changes that are present on the master branch of the |NCS|, as
 Highlights
 **********
 
-* Added support for NFC and *pair before use* type of accessories to the Apple Find My add-on.
+There are no entries for this section yet.
```

Comment:
```suggestion
nRF Connect for Cloud is now nRF Cloud.
```

Reply:
Highlights are populated by the SWGMs and PMTs, so I've only applied the change to the documentation section.